wandb/apis/public/api.py

```diff
@@ -680,14 +680,22 @@ def from_path(self, path):
         )
 
     def _parse_project_path(self, path):
-        """Return project and entity for project specified by path."""
+        """Return project, entity, and organization for project specified by path."""
         project = self.settings["project"] or "uncategorized"
         entity = self.settings["entity"] or self.default_entity
         if path is None:
-            return entity, project
+            return "", entity, project
         parts = path.split("/", 1)
         if len(parts) == 1:
-            return entity, path
+            return "", entity, path
+        if len(parts) == 2:
+            entity_part, project = parts
+            org, entity = (
+                (entity_part, entity)
+                if is_artifact_registry_project(project)
+                else ("", entity_part)
+            )
+            return org, entity, parts[1]
         return parts
```

Comment:
should the default return be `return "", *parts` because you have to return the org name too?

Reply:
not sure if org name should be returned by this function at all though